src/main/java/org/intellij/sdk/codesync/repoManagers/DeletedRepoManager.java

```diff
@@ -60,9 +70,15 @@ public void copyFiles(String[] filePaths) {
     */
     public void copyFiles(String[] filePaths, String projectRepoPath) {
         for (String filePath: filePaths) {
+            String originalFilePath = filePath;
+
+            if (Utils.isWindows()) {
```

Comment:
I think we need this only when copying within system directories. i.e. from .shadow to .delete. 
When copying from actual repo, we need `:` in the source path.
btw, I might have missed, where are we using `filePath` variable?